climakitae/util/warming_levels.py

```diff
@@ -210,3 +222,131 @@ def drop_invalid_sims(ds, selections):
         )
     )
     return ds.sel(all_sims=valid_sim_list)
+
+
+def generate_ssp_dict() -> dict[str, pd.DataFrame]:
+    """
+    Loads historical and SSP scenario CSVs into one dictionary.
+
+    Returns:
+        Dict[str, pd.DataFrame]: A dictionary mapping scenario names to their
+        pandas DataFrames, indexed by year.
+    """
+    ssp119_data = read_csv_file(ssp119_file, index_col="Year")
+    ssp126_data = read_csv_file(ssp126_file, index_col="Year")
+    ssp245_data = read_csv_file(ssp245_file, index_col="Year")
+    ssp370_data = read_csv_file(ssp370_file, index_col="Year")
+    ssp585_data = read_csv_file(ssp585_file, index_col="Year")
+    hist_data = read_csv_file(hist_file, index_col="Year")
+
+    ssp_dict = {
+        "Historical": hist_data,
+        "SSP 1-1.9": ssp119_data,
+        "SSP 1-2.6": ssp126_data,
+        "SSP 2-4.5": ssp245_data,
+        "SSP 3-7.0": ssp370_data,
+        "SSP 5-8.5": ssp585_data,
+    }
```

Comment:
read directly into the dictionary, and just construct the dictionary in the return statement